checker/worker/worker.go

```diff
@@ -67,7 +68,13 @@ func (worker *Checker) Start() error {
 		worker.Logger.Info("Remote checker disabled")
 	}
 
+	const maxParallelChecksMaxValue = 1024 * 8
```

Comment:
Can you leave a comment why is that const set exactly to that value? 
Also, since `MaxParallelChecks ` is already placed in config, does this const really useful? Wouldn't we like to leave it up to user set whatever they need?

Reply:
> why is that const set exactly to that value?

I used the current value from the config file with a margin